utils/lpac/patches/0001-driver-add-uqmi-backend.patch

```diff
@@ -0,0 +1,347 @@
+From 96c73de212c84caa1cc2796980e762321e0acdc3 Mon Sep 17 00:00:00 2001
+From: David Bauer <david.bauer@uniberg.com>
+Date: Wed, 27 Mar 2024 22:20:16 +0100
+Subject: [PATCH] driver: add uqmi backend
```

Comment:
I couldn't find that it was submitted to lpac repository. Even though, this is OpenWrt specific, you should try to upstream it, because we will care your patch for ages and if there will be conflict every single time, someone from us will need to rebase it.

Reply:
Decision on this is pending and dependent on the progress on uqmid. No need to upstream if the rewrite is months away.

Also i don't think a segregation in "you" and "we" is neither respectful nor adequate regardless of who opened the PR.

Reply:
Same could be said in self merging PRs. ;-)